src/formatting/expr.rs

```diff
@@ -1930,6 +1936,26 @@ pub(crate) fn rewrite_assign_rhs<S: Into<String>, R: Rewrite>(
     rewrite_assign_rhs_with(context, lhs, ex, shape, RhsTactics::Default)
 }
 
+// The left hand side must contain everything up to, and including, the
+// assignment operator.
+pub(crate) fn rewrite_assign_rhs_with_span<S: Into<String>, R: Rewrite>(
+    context: &RewriteContext<'_>,
+    lhs: S,
+    ex: &R,
+    shape: Shape,
+    between_span: Span,
+) -> Option<String> {
+    rewrite_assign_rhs_with_comments(
+        context,
+        lhs,
+        ex,
+        shape,
+        RhsTactics::Default,
+        between_span,
+        true,
+    )
+}
+
```

Comment:
I'm okay with adding another helper function here to avoid repetitive specification of the rhs tactics, but I think we'd probably want to expose as a parameter the flag which controls whether comments are allowed between the assignment op and rhs without forcing line wrapping (i.e. the last arg in the `rewrite_assign_rhs_with_comments` as a param on `rewrite_assign_rhs_with_span` instead of a hardcoded `true`)

Reply:
I removed the helper function and changed the code to read directly `rewrite_assign_rhs_with_comments` instead, as during the evaluation of the multi-lie comments issue I found that using this helper function makes the code less clear.